spec/features/admin_user/admin_edit_spec.rb

```diff
@@ -0,0 +1,66 @@
+require "rails_helper"
+
+RSpec.feature "admin can edit personal details", type: :feature do
+  let(:jon) { User.create(full_name: "Jon Snow",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.